.github/CODEOWNERS

```diff
@@ -101,7 +101,50 @@
 /codecov.yml @lilithxxx
 /core/templates/dev/head/css/.stylelintrc @kevinlee12
 /tox.ini @kevinlee12
-/scripts/ @kevinlee12
+/scripts/install_third_party_libs.py @kevinlee12
+/scripts/pre_push_hook.py @kevinlee12
+/scripts/build.py @kevinlee12
+/scripts/pre_commit_linter.py @kevinlee12
+/scripts/linter_utils_test.py @kevinlee12
+/scripts/clean.py @kevinlee12
+/scripts/deploy.py @kevinlee12
```

Comment:
@ankita240796 Would you mind also jointly, owning this? I'm not too familiar with the code that occurs here.

Reply:
Done.